tests/test_A008589.py

```diff
@@ -0,0 +1,9 @@
+from oeis import A008589
+
+
+def testA008589():
```

Comment:
I bet this test is not found by pytest, as it's lacking an underscore.

Also I can think of another test:
```
@given(integers())
def test_A008589(start):
    assert A008589(start, 1)[0] % 7 == 0
```